waves/wallet/src/wallet.rs

```diff
@@ -249,12 +258,50 @@ pub struct BalanceEntry {
 
 #[derive(Debug)]
 pub struct Wallet {
-    pub name: String,
-    pub encryption_key: [u8; 32],
-    pub secret_key: SecretKey,
+    name: String,
+    encryption_key: [u8; 32],
+    secret_key: SecretKey,
 }
 
+const SECRET_KEY_ENCRYPTION_NONCE: &[u8; 12] = b"SECRET_KEY!!";
```

Comment:
:sob:

Reply:
What do you think about the choice of a static nonce? We could also generate a random number so not every user uses the same one. Thoughts?

Reply:
I was just amused by the string, but it's a good question. I think they ciphertext will always be unique because the plaintext is generated, so it's probably fine.